hw6_task_1.py

```diff
@@ -0,0 +1,47 @@
+def arithmetic(left_operand: int, right_operand: int, operation):
+    """
+            Apply arithmetic operation for provided left and right operands
+
+    """
+
+    if operation == "+":
+        return left_operand + right_operand
+    if operation == "-":
+        return left_operand - right_operand
+    if operation == "*":
+        return left_operand * right_operand
+    if operation == "/":
+        return left_operand / right_operand
+    else:
+        error_message = f"Not known operation: {operation}"
+        return error_message
+
+# final = arithmetic(left_operand=6, right_operand=5, operation="*")
+# print(final)
+
+if __name__ == "__main__":
+    assert arithmetic(3, 4, operation="*") == 12
+    assert arithmetic(3, 4, operation="+") == 7
+    assert arithmetic(25, 5, operation="/") == 5
+    assert type(arithmetic(25, 5, operation="/")) == float
+    assert arithmetic(5, 5, operation="//") == f"Not known operation: //"
+    c = 0
+    assert arithmetic.__doc__ == """
+            Apply arithmetic operation for provided left and right operands
+
+    """
```

Comment:
why do you change my code? 

Your task is only to implement the function. code in if-> main should remains as it was